site/content/how-to/traffic-management/securing-backend-traffic.md

```diff
@@ -12,9 +12,15 @@ Learn how to encrypt HTTP traffic between NGINX Gateway Fabric and your backend
 
 In this guide, we will show how to specify the TLS configuration of the connection from the Gateway to a backend pod/s via the Service API object using a [BackendTLSPolicy](https://gateway-api.sigs.k8s.io/api-types/backendtlspolicy/). This covers the use-case where the service or backend owner is doing their own TLS and NGINX Gateway Fabric needs to know how to connect to this backend pod that has its own certificate over HTTPS.
 
+## Note on Gateway API Experimental Features
+
+{{< important >}} BackendTLSPolicy is a Gateway API resource from the experimental release channel. {{< /important >}}
```

Comment:
I feel `note` is more appropriate to use here

```suggestion
{{< note >}} BackendTLSPolicy is a Gateway API resource from the experimental release channel. {{< /note >}}
```

Reply:
Down to make the change, but I'll wait on docs-team input